lua/telescope/pickers.lua

```diff
@@ -1,6 +1,7 @@
 require "telescope"
 
 local a = vim.api
+local if_nil = vim.F.if_nil
```

Comment:
Small nit, just leave `vim.F.if_nil` for now. We'll migrate from `utils.get_default` to internal functions eventually. Therefore better not to introduce shorthands for now here.